include/macros.h

```diff
@@ -17,6 +17,7 @@
 
 #define SQ(x) ((x)*(x))
 #define ABS(x) ((x) >= 0 ? (x) : -(x))
+#define ABS_ALT(x) ((x) < 0 ? -(x) : (x))
```

Comment:
I'm not sure this macro is worth adding, or maybe define it for `code_800EC960.c` only?
apart from that file with 3 uses there is a single spot in camera that could use it, which isn't much use imo

Reply:
Its defined in MM where it is needed in some places.  It might be useful for retail.

Reply:
I'm not particularly attached to this macro, I just thought it'd be more useful. Maybe that's just because most of my time has been on either `code_800EC960.c` or `camera.c` so I was getting familiar with it. If it's really limited to just those two files, I don't see a strong need to have it in tbh. We can always add it in later if it turns out retail does use it more often for whatever reason. Hard to say if it actually will be needed or not

Reply:
My reasoning is:
- It is not highly useful to help making code readable (or if it is, then it can just defined in the file where it is used)
- It has no use in writing new code since the equivalent `ABS` macro already exists
- One can find other `ABS` variants, like `x <= 0 ? -x : x`, so adding such variants opens a can of worms imo

---------

I don't see an issue with defining it for `code_800EC960.c` though (and I would leave the single one in z_camera not using a macro)

There actually are strong clues this macro does exist and is used a bunch more than what you changed in `code_800EC960`, for example:

https://github.com/zeldaret/oot/blob/94502725030b2778d8f15694e10499f76d69090f/src/code/code_800EC960.c#L4342-L4346

https://github.com/zeldaret/oot/blob/94502725030b2778d8f15694e10499f76d69090f/src/code/code_800EC960.c#L4616-L4620

you may want to just replace existing `ABS` for that file (`#undef ABS` `#define ABS...`) since the `ABS` with `>= 0` isn't used in there

Could this be one more clue towards the theory that audio was built as a separate library? Is this file part of what would be such a library